dist/stat_data.py

```diff
@@ -397,6 +397,27 @@ def __init__(self, name, desc, flags=''):
     HistoryStat('hs_gc_pages_evict', 'history pages added for eviction during garbage collection'),
     HistoryStat('hs_gc_pages_removed', 'history pages removed for garbage collection'),
     HistoryStat('hs_gc_pages_visited', 'history pages visited for garbage collection'),
+    HistoryStat('hs_write_addr_prepared', 'page addresses written that included at least one prepare state'),
+    HistoryStat('hs_write_addr_newest_start_durable_ts', 'page addresses written that include a newest start durable timestamp '),
+    HistoryStat('hs_write_addr_newest_stop_durable_ts', 'page addresses written that include a newest stop durable timestamp '),
+    HistoryStat('hs_write_addr_oldest_start_ts', 'page addresses written that include a oldest start timestamp '),
+    HistoryStat('hs_write_addr_oldest_start_txn', 'page addresses written that include a oldest start transaction ID '),
+    HistoryStat('hs_write_addr_newest_stop_ts', 'page addresses written that include a newest stop timestamp '),
+    HistoryStat('hs_write_addr_newest_stop_txn', 'page addresses written that include a newest stop transaction ID'),
```

Comment:
Not a big fan of "page addresses", maybe "internal page"?

Don't feel strongly.